.github/workflows/changelog.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 env:
   # Also change CACHE_VERSION in the other workflows
-  CACHE_VERSION: 24
+  CACHE_VERSION: 26
```

Comment:
Why are we bumping this again?

Reply:
Because of astroid 2.12.8 and 2.12.9. (``25`` is already taken by ``2.12.8``)

Reply:
Oh huh, I didn't notice `main` isn't on `2.12.9` yet. How did that happen?

Reply:
I upgraded astroid on the maintenance branch directly instead of doing it on main and then backporting (as we were going go merge the maintenance branch in main anyway).